pkg/csi/cinder/controllerserver.go

```diff
@@ -110,36 +112,66 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}
 	content := req.GetVolumeContentSource()
 	var snapshotID string
-	var sourcevolID string
+	var sourceVolID string
+	var sourceBackupID string
+	var backupsAreEnabled bool
+	backupsAreEnabled, err = cloud.BackupsAreEnabled()
```

Comment:
I think the added delay for a createVolume or createSnapshot action should not be noticeable. Also if we do this at controller startup it might be hard to debug when adding backups capability to an existing cluster.

Reply:
This can be done as a second step, I think.